demo/chatbot_multimodal/run.py

```diff
@@ -16,18 +20,21 @@ def add_message(history, message):
     return history, gr.MultimodalTextbox(value=None, interactive=False)
 
 def bot(history):
-    response = "**That's cool!**"
-    history[-1][1] = ""
-    for character in response:
-        history[-1][1] += character
-        time.sleep(0.05)
-        yield history
-
-with gr.Blocks() as demo:
+    history[-1][1] = "Cool!"
+    return history
+
+fig = random_plot()
+
+with gr.Blocks(fill_height=True) as demo:
     chatbot = gr.Chatbot(
-        [],
+        [["Image", gr.Image(value="files/avatar.png", render=False)],
+         ["Video", gr.Video(value="files/world.mp4", render=False)],
+         ["Audio", gr.Audio(value="files/cantina.wav", render=False)],
+         ["Plot", gr.Plot(value=fig, render=False)],
+         ["Gallery", gr.Gallery(value=["files/lion.jpg", "files/cheetah.jpg", "files/zebra.jpg"], render=False)]],
```

Comment:
@freddyaboulton Any suggestions on how to simplify this so `render=False` is not necessary? I tried passing it as a constructor arg during postprocessing, but the component would still render.

Reply:
Can you call `unrender` for each component in the initial value in the chatbot init?

Reply:
Wondering if it would work.